packages/web/src/components/Shelves/FlowerItem/FlowerItem.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FlowerItem } from '../FlowerItem';
```

Comment:
Git suggest next changes:
Useless path segments for "../FlowerItem", should be "./"